package.json

```diff
@@ -30,6 +30,9 @@
     "./worker": {
       "types": "./build/worker.d.ts",
       "default": "./build/worker.js"
+    },
+    "./worker.global": {
```

Comment:
i have no idea what would be proper name for this 🤷 , `worker.global.js` is just something that `tsup` builds by default if target is `iife`

Reply:
Perhaps `@carto/api-client/worker-compat`?

I've seen this `*-compat` naming used for "compatibility-mode" builds of WASM libraries, using inline base64 instead of binaries. I think the same intent describes what we're doing here, and this way we could change the implementation (if needed) without renaming.

Reply:
Done,

Bundle is `build/worker-compat.js` and exists in `exports` as `worker-compat`.